packages/libs/react-ui/src/components/TextAreaField/TextAreaField.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ITextAreaFieldProps } from './TextAreaField';
```

Comment:
In stories, I usually import from @component because it acts as an additional check that the export was done correctly. This essentially tests it while importing it like a consuming application